### DISCUSSION TITLE
How to disable signup in @supabase/auth-ui-react?

### DISCUSSION THREAD

**Discussion by danaoairuike** [supabase/supabase#14496] [Questions]
the app is just for internal use. So, we need to disable signup. How ? Thanks

**Comment by deviprsd** [comment]
+1

**Comment by tsanyqudsi** [comment]
it's written here https://github.com/supabase/auth-ui/issues/47#issuecomment-1301533343

`<Auth
  supabaseClient={supabase}
  view="sign_in"
  showLinks={false}
/>`
